rfc-005-template-contributing-process.md

```diff
@@ -0,0 +1,154 @@
+- John Davenport
+- Meg Albert
+- Ryan Macklin
+- Sachin Karol
+
+
+## Proposal overview
+
+This RFC outlines the proposed process for contributing templates to the Good Docs Project. This proposal explains:
+
+- The workflow template contributors will follow as they begin writing a template until it is fully merged within the `templates` repository.
+- The documents (deliverables) that are required for each template set and which are optional.
+- Our project's template quality requirements. (Note that these quality guidelines will likely evolve over time.)
+- The roles and resources our project needs to create in order to best support template contributors as they submit templates to our project.
+
+Note that project management logistics (such as whether template projects are represented as epics or issues, how template contributors indicate the status of their template set, how templates are planned for releases, etc.) are out of scope for this proposal and will be worked out later.
+
+
+## Motivation
+
+Creating templates is core to the mission of the Good Docs Project.
+This proposed template contributing process will ensure that our contributors have clear processes, guidelines, and resources as they contribute templates to our project.
+
+
+## Proposal
+
+In order to view this proposal in-depth, see these files that are included in this pull request (in this order):
+
+- template-contributing-guide.md
+- template-deliverables.md
+- template-quality-checklist.md
+- template-roles-and-resources.md
+
+I'll provide a few key highlights here:
+
+Template contributors will write templates as participants in template writing working groups that meet weekly and are led by an experienced template mentor.
+These template mentors will communicate regularly with the templates coordinator, who oversees the entire template process as a project manager.
+Additionally, a large, inclusive group of volunteer community reviewers will review templates that are in the workshop phase.
+A small group of experienced pull request reviewers will review and check the templates for quality before merging them into the `templates` repository.
+
+Templates will be written in six different phases with certain checkpoints and milestones that each template must meet along the way:
+
+1. **Join the community** - A new contributor joins the community and find an initiative they want to work on.
+
+2. **Adopt a template** - If they are interested in contributing templates, they join a template writing working group led by a template mentor. They decide which template set to begin working on.
+
+3. **Research and draft** - They research best practices and look at examples of the type of document they are working on. When they are finished, they will create a rough draft.
+
+4. **Workshop the drafts with the community** - Revise and refine their template set with the community pull reviewers. At least three community reviews are required to move to the next phase.

```

Comment:
Can we reword Workshop the drafts to Review the drafts.  My understanding is anybody in the community can review the draft. Why only community pull reviewers? What do you mean by three community reviews? Do you mean 3 rounds of review....Wondering if we can provide information on how the review happens..

Reply:
I agree that to make this more clear, we could just say "Revise drafts based on community review". At least three community reviews sounds to me like at least three people need to review, not three rounds of review, but I agree it is unclear. Three rounds of review seems like an excessively high bar. As for how the review happens, I would assume it is simply reading the drafts and leaving comments, not likely something in real-time.

Reply:
My impression is that "Workshop" refers to actually using the template to create documents, not reviewing the actual template. My assumption is that the reviewing will happen in 1-3, and that "workshopping" is more than just reviewing and providing feedback.

Reply:
Yes, for clarification, it's not 3 rounds of reviews. It's that at least 3 different community reviewers will provide feedback on the draft. 3 community reviewers are necessary to advance to the next stage.

Can anyone suggest a simple, succinct verb to replace "workshop" that encompasses the meaning here?

Reply:
Thanks to everyone who commented on this part of the RFC. We had a lengthy discussion about it at our last general meeting. The consensus was that the term "workshop," although accurate, was not necessarily understood across all cultures. For that reason, we're going to change the name of this phase to "Get feedback on drafts from the community." There was also some confusion about who could review these drafts. We'll clarify that community reviewers includes anyone who is a member of the Good Docs Project, including members of template writing working groups. For now, we'll keep the requirement to have at least three community reviews are required to move to the next phase. We'll adjust this requirement if we discover it is too difficult.